### PR TITLE
Added nullable, and file-scoped namespaces.

### PR DESCRIPTION
**Pull request by Haplois**
- Adds `nullable`
- Adds file-scoped namespaces.

Part of #164

### CODE REVIEW

**Comment by Haplois** [comment]
@DamianEdwards, can you review this?

File: template_feed/Microsoft.DotNet.Test.ProjectTemplates.6.0/content/MSTest-CSharp/.template.config/template.json [hunk 1]
```diff
@@ -36,6 +36,19 @@
       "replaces": "net6.0",
       "defaultValue": "net6.0"
     },
+    "": {
```
Comment (DamianEdwards) [inline template_feed/Microsoft.DotNet.Test.ProjectTemplates.6.0/content/MSTest-CSharp/.template.config/template.json:39]: is this missing the identifier?

File: template_feed/Microsoft.DotNet.Test.ProjectTemplates.6.0/content/MSTest-CSharp/.template.config/template.json [hunk 2]
```diff
@@ -36,6 +36,19 @@
...
+      "type": "parameter",
+      "datatype": "text",
+      "description": "Sets the LangVersion property in the created project file",
+      "defaultValue": "",
+      "replaces": "$(ProjectLanguageVersion)"
+    },
+    "nullable": {
```
Comment (DamianEdwards) [inline template_feed/Microsoft.DotNet.Test.ProjectTemplates.6.0/content/MSTest-CSharp/.template.config/template.json:46]: Was there a specific reason you added this and langVersion parameters? We decided to wait for feedback before doing it for the other templates.

**Review by DamianEdwards — COMMENTED** [review]
Note the file-scoped namespaces support in the compiler just landed and hasn't made it into the SDK just yet (it's still waiting to be merged to relevant compiler branch [here](https://github.com/dotnet/roslyn/pull/54763)).

File: template_feed/Microsoft.DotNet.Test.ProjectTemplates.6.0/content/MSTest-CSharp/.template.config/template.json (hunk 2 above)
Comment (Haplois) [reply template_feed/Microsoft.DotNet.Test.ProjectTemplates.6.0/content/MSTest-CSharp/.template.config/template.json:46]: Ah thanks. Removing.

**Review by DamianEdwards — APPROVED** [review]

**Pull request merged**